src/Core/ObjectRegistry.php

```diff
@@ -66,25 +66,27 @@
      * @param array $config Additional settings to use when loading the object.
      * @return mixed
      */
-    public function load($objectName, $config = [])
+    public function load($objectName, array $config = [])
```

Comment:
Actually this is bugging me too, please take a look at `_checkDuplicate` method.
`
        unset($config['enabled'], $existingConfig['enabled']);

        $fail = false;
        foreach ($config as $key => $value) {
`